repo_e/sum_words_string.py

```diff
@@ -0,0 +1,12 @@
+def sum_word_in_string(string):
```

Comment:
overall, nice approach, you can also try to use `Counter` from `collections` library and see how it simplifies your solution

Reply:
I did it